react-native-community/react-native-viewpager
# viewpager2

# Summary

I migrated whole ViewPager  to version 2. All provided api should work after migration. 
This is alpha version. I needed to copy and paste whole VP2 from the google source, because there was a problem with RecyclerView and I needed to fix it.  It means, that we need to maintenance it on our own once we want to bump the version of VP2.  

Close https://github.com/react-native-community/react-native-viewpager/issues/118

## Test Plan
* Check example
* Check this migration in your existing app

I did some tests on: 
* Samsung S8 
* Moto X4 
* Xiaomi RedMi 5A

And all API works fine without any breaks

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ❌     |
| Android |    ✅     |

## Checklist
